doc/manual/manual.tex

```diff
@@ -10666,6 +10667,71 @@ \subsubsection{Material models}
+
+Throughout this manual and within the code of \aspect{}, pressure and temperature are treated as independent thermodynamic variables, and so we here express the physical properties in terms of the pressure and temperature derivatives of the specific Gibbs free energy $\mathcal{G}^{*}(p, T)$ [\textit{J/kg}] (the asterisk indicates that this is a specific quantity, rather than a molar one):
+\begin{eqnarray}
+  S^{*} &=& -\pddc{\mathcal{G}^{*}}{T}{p} \\
+  \frac{1}{\rho} &=& \pddc{\mathcal{G}^{*}}{p}{T} \label{eq:mm_density} \\
+  \frac{\alpha}{\rho} &=& \pddsqc{\mathcal{G}^{*}}{p}{T} \label{eq:mm_alpha_g} \\
+  \beta_T &=& -\rho \pddsq{\mathcal{G}^{*}}{p}{T} \label{eq:mm_betaT_g} \\
+  C_p^{*} &=& -T \pddsq{\mathcal{G}^{*}}{T}{p} \label{eq:mm_isobaric_heat_capacity} \\
+  \beta_S &=& \beta_T - \frac{\alpha^2 T}{\rho C_p^{*}} \label{eq:mm_isentropic_compressibility} \\
+  \frac{C_V^{*}}{C_p^{*}} &=& \frac{\beta_S}{\beta_T} \\
+  \gamma &=& \frac{\alpha }{\beta_T \rho C_V^{*}} 
+\end{eqnarray}
+\noindent where $S^{*}$ is the specific entropy, $C_P^{*}$ and $C_V^{*}$ are the specific isobaric and isenchoric heat capacities, $\beta_T$ and $\beta_S$ are the isothermal and isotropic compressibilities, and $\gamma$ is the thermodynamic Gr\"{u}neisen parameter. The subscript indicates the thermodynamic variable ($P$ or $T$) that is held constant.
+
+
+Thermodynamically self-consistent material models must obey the explicit and implicit relations between the different properties \emph{at all pressures and temperatures}:
+
+\paragraph{Consistency in $\boldsymbol{\rho}$-$\boldsymbol{\alpha}$ and $\boldsymbol{\rho}$-$\boldsymbol{\beta_T}$}
+Using the chain rule to combine~\eqref{eq:mm_density},~\eqref{eq:mm_alpha_g} and~\eqref{eq:mm_betaT_g} yields the more familiar definitions of $\alpha$ and $\beta_T$:
+\begin{eqnarray}
+  \alpha &=& -\frac{1}{\rho} \pddc{\rho}{T}{p} \label{eq:mm_thermal_expansivity} \\
+  \beta_T &=& \frac{1}{\rho} \pddc{\rho}{p}{T} \label{eq:mm_isothermal_compressibility}
+\end{eqnarray}
+
+\paragraph{Isobaric heat capacity}
+
+We start by taking the partial derivative of the isobaric heat capacity~\eqref{eq:mm_isobaric_heat_capacity} with respect to pressure at constant temperature:
+\begin{eqnarray}
+  \pddc{C_p^{*}}{p}{T} &=& -T \pddcbc{\mathcal{G}^{*}}{T}{p} \\
+  &=& -T\pddc{\left(\alpha / \rho \right)}{T}{p} \label{eq:heat_capacity_p_dependence}
+\end{eqnarray}
+From this expression it becomes clear that if $\alpha / \rho$ has any temperature dependence, the heat capacity \emph{cannot} be globally constant. One way to solve this issue is to define heat capacity at constant pressure, and then integrate~\eqref{eq:heat_capacity_p_dependence} with respect to pressure:
+\begin{equation}
+  C_p^{*}(p, T) = C_p^{*}(p_{\textrm{ref}}, T) + \int_{p_{\textrm{ref}}}^p \pddc{C_p^{*}}{p}{T} dp
```

Comment:
The equation you reference here (101) is not the one that is written down (100).

Reply:
Also, does that expression get any simpler if you actually put it in?